EDU-18313 Payment section tabs articles - Create_Update (#2655)
vtexdocs/dev-portal-content: docs/guides/Payments/payments-checkout-ui-category.md

---
title: "Payment App and Checkout UI"
slug: "payments-checkout-ui-category"
hidden: false
createdAt: "2026-05-13T00:00:00.000Z"
updatedAt: "2026-05-13T00:00:00.000Z"

> excerpt: "Build custom payment experiences using Payment Apps and customize the Smart Checkout payment layout."

Reviewer: ```suggestion
excerpt: "Build custom payment experiences using Payment Apps and customize the SmartCheckout payment layout."
```

---
VTEX supports custom payment interfaces through two approaches:

- **Payment App:** build a custom payment UX using VTEX IO that runs within the checkout. This enables providers to offer payment experiences beyond the standard transparent or redirect flows.

> - **Checkout UI customization:** customize the layout and display of payment methods in the VTEX Smart Checkout.

Reviewer: ```suggestion
- **Checkout UI customization:** customize the layout and display of payment methods in the VTEX SmartCheckout.
```

## In this section

| Article | Description |
| --- | --- |
| [Payment App](https://developers.vtex.com/docs/guides/payments-integration-payment-app) | How to build a Payment App on VTEX IO for custom payment experiences at checkout. |

> | [Layout Development Guide for payment methods in Smart Checkout VTEX](https://developers.vtex.com/docs/guides/layout-development-guide-for-payment-methods-in-smart-checkout-vtex) | How to develop custom layouts for payment methods in Smart Checkout. |

Reviewer: ```suggestion
| [Layout Development Guide for payment methods in SmartCheckout VTEX](https://developers.vtex.com/docs/guides/layout-development-guide-for-payment-methods-in-smart-checkout-vtex) | How to develop custom layouts for payment methods in SmartCheckout. |
```
